src/main/java/com/example/hop_oasis/controller/SnackController.java

```diff
@@ -0,0 +1,82 @@
+package com.example.hop_oasis.controller;
+
+import com.example.hop_oasis.dto.SnackDto;
+import com.example.hop_oasis.dto.SnackImageDto;
+import com.example.hop_oasis.dto.SnackInfoDto;
+import com.example.hop_oasis.service.data.SnackImageServiceImpl;
+import com.example.hop_oasis.service.data.SnackServiceImpl;
+import jakarta.servlet.annotation.MultipartConfig;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("snacks")
+@MultipartConfig
+public class SnackController {
+    private final SnackServiceImpl snackService;
+    private final SnackImageServiceImpl imageService;
+
+    @GetMapping
+    public ResponseEntity<List<SnackInfoDto>> getAllSnacks(){
+        return ResponseEntity.ok().body(snackService.getAllSnacks());
+    }
+    @PostMapping
+    public String save(@RequestParam("name") String name,
+                       @RequestParam("weight_large") double weightLarge,
+                       @RequestParam("weight_small") double weightSmall,
+                       @RequestParam("price_large") double priceLarge,
+                       @RequestParam("price_small")double priceSmall,
+                       @RequestParam("description") String description,
+                       @RequestParam("image") MultipartFile image){
```

Comment:
you either use camelCase as for `beerId` request parameter, or snake_case.
it's crucial to keep your API consistent